newIDE/app/src/Utils/Analytics/EventSender.js

```diff
@@ -108,22 +121,10 @@ export const identifyUserForAnalytics = (
     hearFrom: profile ? profile.hearFrom : undefined,
   };
 
-  // If the user is logged in, alias the user with its firebase id.
-  if (firebaseUser) {
-    const aliasKey = firebaseUser.uid + '#' + getUserUUID();
-    if (!posthogAliasMade.includes(aliasKey)) {
-      // Try to Alias the random UUID to the Firebase ID.
-      // 2 cases:
-      // - Either it's the first time the user logs in and posthog does not know
-      //   about the Firebase ID, in which case we specify the firebaseId as 2nd parameter
-      //   of the alias method.
-      // - Or the user has already logged in and posthog knows about the Firebase ID,
-      //   in which case PostHog will reject the merging of the 2 users.
-      //   We then need to ensure we identify the user by its Firebase ID.
-      posthog.alias(getUserUUID(), firebaseUser.uid);
-      posthogAliasMade.push(aliasKey);
-    }
-  }
+  // Important. We never alias the user while identifying, to avoid
+  // linking the Firebase ID as the future reference of possible new UUIDs.
+  // This allows keeping the Firebase ID as the reference of the first Posthog
+  // account that was linked on signup.
```

Comment:
I don't really understand this ("as the future reference" and how not doing something is allowing "keeping the firebase id as the reference of the first Posthog account").